coreneuron/gpu/nrn_acc_manager.cpp

```diff
@@ -856,10 +835,10 @@ void nrn_newtonspace_copyto_device(NewtonSpace* ns) {
     pd = (double*)acc_copyin(ns->rowmax, n * sizeof(double));
     acc_memcpy_to_device(&(d_ns->rowmax), &pd, sizeof(double*));
 
-    pint = (int*)acc_copyin(ns->perm, n * sizeof(int));
+    int* pint = (int*)acc_copyin(ns->perm, n * sizeof(int));
```

Comment:
I usually avoid duplicating the type

```suggestion
    auto pint = static_cast<int*>(acc_copyin(ns->perm, n * sizeof(int)));
```